### PR TITLE
cache virtual env on travis

### PR DESCRIPTION
**Pull request by snopoke**
Re-attempt of https://github.com/dimagi/commcare-hq/pull/8590

### CODE REVIEW

**Comment by snopoke** [comment]
:white_check_mark: 
image

**Comment by snopoke** [comment]
@dannyroberts

**Comment by benrudolph** [comment]
woah cool

File: .travis.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ sudo: false
 cache:
   directories:
     - $HOME/.cache/pip
+    - $HOME/virtualenv/python$TRAVIS_PYTHON_VERSION.9
```
Comment (benrudolph) [inline .travis.yml]: do you know why this works and what danny tried didn't?

**Comment by czue** [comment]
:racehorse:

**Comment by snopoke** [comment]
I don't know why. I got this by looking at the datadog .travis file.

On 30 October 2015 at 16:32, Cory Zue <notifications@github.com> wrote:

> [image: :racehorse:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/8860#issuecomment-152541813>.
>



-- 
Simon Kelly
Senior Engineer | Dimagi South Africa

**Comment by dannyroberts** [comment]
Awesome!! :speedboat:

**Pull request merged**